ci-operator/step-registry/openshift/microshift/infra/iso-boot/openshift-microshift-infra-iso-boot-commands.sh

```diff
@@ -93,8 +95,8 @@ set -xeuo pipefail
 cd ~/microshift
 VM_IP=\$(./scripts/devenv-builder/manage-vm.sh ip -n ${VM_NAME})
 
-timeout 5m bash -c "until ssh redhat@\${VM_IP} hostname; do sleep 5; done"
-timeout 5m bash -c "date; until ssh redhat@\${VM_IP} \"sudo systemctl status greenboot-healthcheck | grep 'active (exited)'\"; do sleep 5; done; date"
+timeout 8m bash -c "until ssh redhat@\${VM_IP} hostname; do sleep 5; done"
+timeout 8m bash -c "date; until ssh redhat@\${VM_IP} \"sudo systemctl status greenboot-healthcheck | grep 'active (exited)'\"; do sleep 5; done; date"
```

Comment:
This makes me wonder if we should use beefier VMs (since there is enough resources) to shorten everything, and have a small "minimum requirements test" that runs fast enough to validate the 2 core, 2G are still ok.

Reply:
That could make sense, yeah. I'm not sure if creating the VMs is limited by CPU, memory, or I/O.